plugins/scaffolder/src/types.ts

```diff
@@ -51,24 +38,100 @@ export type ListActionsResponse = Array<{
   id: string;
   description?: string;
   schema?: {
-    input?: JSONSchema;
-    output?: JSONSchema;
+    input?: JSONSchema7;
+    output?: JSONSchema7;
```

Comment:
in some other places we've made this just `unknown`, to not put the burden on the callers to know of, and import, and get the types right for, json schemas.

Reply:
No strong opinion on my part though, just a note

Reply:
But these are actually going to be written as JSON Schema in typescript, so I think the type makes sense here.

Reply:
Yeah but you can do that on the caller side. Since these are really deep, complex types, you know how it is when the caller is on a slightly different version than you, or use a different json schema aware library together with our stuff, and you get a red squiggly because deeeep down in some structure they are very slightly different in their types, in a place that you don't use :)

Reply:
But again, no biggie, leave this as is. I'm just here for the debate :P